### ISSUE TITLE
Changing from Favorites to All

### ISSUE THREAD

**Issue by thomastthai** [driftyco/ionic-conference-app#124]
When viewing Favorites and clicking on All button at the top, it won't show all the events again. The click is registered though. 

![image](https://cloud.githubusercontent.com/assets/16532581/12099963/bbe80e1e-b2e0-11e5-9a79-f0eb18ae9255.png)

**Comment by brandyscarney** [comment]
Have you pulled the latest code? Running `ionic info` should give you this:

```
Ionic Version: 2.0.0-alpha.46
Ionic CLI Version: 2.0.0-beta.15
```

I updated to 46 this morning and it should be working.

**Comment by thomastthai** [comment]
Here is my current set up. I will pull latest and test again. Thank you for the quick reply.

C:\Users\Thomas\ionic2\employeeDirectory>ionic info

Your system information:

Cordova CLI: 5.4.1
Gulp version:  CLI version 3.9.0
Gulp local:
Ionic CLI Version: 2.0.0-beta.15
Ionic App Lib Version: 2.0.0-beta.8
OS:
Node Version: v4.2.4

**Comment by brandyscarney** [comment]
Make sure to run `ionic info` from within the project folder to get the Ionic Version. :smile:

**Comment by thomastthai** [comment]
Updated ionic using: npm install -g ionic@beta

C:\Users\Thomas\ionic2\ionic-conference-app>ionic info

Your system information:

Cordova CLI: 5.4.1
Gulp version:  CLI version 3.9.0
Gulp local:
Ionic Version: 2.0.0-alpha.45
Ionic CLI Version: 2.0.0-beta.15
Ionic App Lib Version: 2.0.0-beta.8
OS:
Node Version: v4.2.4

It still won't show all:

![image](https://cloud.githubusercontent.com/assets/16532581/12101160/1802fb58-b2e8-11e5-8f35-6763e0a29e50.png)

**Comment by brandyscarney** [comment]
So running `npm install -g ionic@beta` updates the Ionic **CLI** which would affect the commands you run in the terminal for Ionic e.g. `ionic serve`, `ionic start`, etc. and this version shows here: `Ionic CLI Version: 2.0.0-beta.15`

If you want to update the framework itself in an existing project you would run `npm install ionic-framework@latest --save` for the latest version or `npm install ionic-framework@2.0.0-alpha.46 --save` for an exact version. In this case the conference app has already been updated to the latest version so you just need to run `npm install` from within the project folder and this will install all of the npm packages declared in the `package.json` file in your `node_modules` folder. 

Here is a general overview: http://ionicframework.com/docs/v2/resources/using-npm/

TL;DR: please try running `npm install` in the project directory.

**Comment by thomastthai** [comment]
Thank you for the feedback Brandy. I grabbed the latest repo for this app and updated ionic-framework and the issue is resolved.
